### ISSUE TITLE
404 Error when script clicks link.

### ISSUE THREAD

**Issue by amphoterism** [vogler/free-games-claimer#28]
```
C:\Users\nick\Downloads\free-games-claimer\update>node epic-games
userAgent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.83 Sa
fari/537.36
Signed in.
Number of free games: 2
urlSlug /en-US/p/runbow-fc9fa4
locator.innerText: Timeout 20000ms exceeded.
=========================== logs ===========================
waiting for selector "//button[@data-testid="purchase-cta-button"][not(contains(.,"Loading"))] >> nth=0"
============================================================
    at file:///C:/Users/nick/Downloads/free-games-claimer/update/epic-games.js:83:124 {
  name: 'TimeoutError'
}
```

**Comment by vogler** [comment]
Should be fixed by [edb90fe](https://github.com/vogler/free-games-claimer/commit/edb90fe9f846fe3e40dd38131d445eadee20a76a). Please reopen if you still encounter the issue after updating.

**Comment by amphoterism** [comment]
That fixed it!
